test/cypress/support/component.js

```diff
@@ -67,6 +71,21 @@ Cypress.Commands.add('mount', (component, options = {}) => {
   options.global = options.global || {};
   options.global.plugins = options.global.plugins || [];
 
+  // create router if one is not provided
+  if (!options.router) {
+    options.router = createRouter({
+      routes: routes,
+      history: createMemoryHistory(),
+    });
+  }
+
+  // Add router plugin
+  options.global.plugins.push({
+    install(app) {
+      app.use(options.router);
```

Comment:
You duplicate code here (check the line 93).  Replace line 97 with line 85  `app.use(options.router);`.

Reply:
I see. Fixed.